bin/PrintGCStats

```diff
@@ -805,6 +807,19 @@ function writePlotData(tstamp, value, widx) {
 }
 
 function recordStats(idx, value) {
+  rs_tstamp = -1;
+  if (available_plot_count != 0 && (idx in plot_idx || (totgc_idx in plot_idx && idx < totgc_idx))) {
+    # Plotting enabled, need to find timestamp on line
+    if (datestamps == 1) {
+      rs_tstamp = getDateTimeStamp();
+    } else {
+      rs_tstamp = getTimeStamp();
+    }
+  }
+  recordStatsTimestamp(idx, value, rs_tstamp);
```

Comment:
Can you clarify why you restructure the code into recordSrarsTimestamp and recordStats()? Are you trying to cover a specific log record by having recordStatsTimestamp?